spyder/plugins/pylint/plugin.py

```diff
@@ -82,7 +82,8 @@ def register(self):
 
         # Connect to Editor
         widget.sig_edit_goto_requested.connect(editor.load)
-        editor.sig_editor_focus_changed.connect(self._set_filename)
+        editor.sig_editor_focus_changed.connect(
+            lambda x, y: self._set_filename)

```

Comment:
```suggestion
            lambda x, y: self._set_filename())
```

I think this is also required (not really sure).